### PR TITLE
Log Analysis Update - Display RuleId instead of changed AlertId

### LINKED ISSUE #344: Log Analysis -> Alerts page should display new fields
### Description

The Alerts Page currently displays the AlertID of an alert, but not enough context about the alert. 
We are going to switch to use UUIDs for the AlertID and when this happens there will be little information on the page regarding the alert. 

### Acceptance Criteria

- AlertID is no longer displayed in the page
- Alert Title is displayed instead
- Users can still natigate from the List Alert page to the Alert details page and back.

### PR DESCRIPTION
**Pull request by alexmylonas**
## Background

The Alerts Page currently displays the AlertID of an alert, but not enough context about the alert.
We are going to switch to use UUIDs for the AlertID and when this happens there will be little information on the page regarding the alert.

Closes #344 

## Changes

- Changed display of AlertId to RuleId (Alert Title is not supported by backend yet)
- Added severity to AlertDetails

## Testing

- Locally
- Dev enviroment

### CODE REVIEW

File: api/graphql/schema.graphql [hunk 1]
```diff
@@ -113,6 +113,8 @@ type LogIntegrationHealth {
 type AlertDetails {
   alertId: ID!
   ruleId: ID
+  title: String
```
Comment (kostaspap) [inline api/graphql/schema.graphql]: Is this placeholder for the future?

Comment (alexmylonas) [reply api/graphql/schema.graphql]: Correct! Do you think I should remove it?

Comment (kostaspap) [reply api/graphql/schema.graphql]: I think it's better to remove yes. I'd prefer not to have dead code in our codebase. It seems fairly easy to add when needed.

Comment (3nvi) [reply api/graphql/schema.graphql]: Yeah, remove what's un-necessary right now. Also, don't forget to add `!` at the fields that **can never be `null`**

File: api/graphql/schema.graphql [hunk 2]
```diff
@@ -128,11 +130,12 @@ type ListAlertsResponse {
 
 type AlertSummary {
   alertId: String!
+  title: String
```
Comment (3nvi) [inline api/graphql/schema.graphql]: same comment here

File: web/src/pages/AlertDetails/graphql/alertDetails.generated.ts [hunk 3]
```diff
@@ -48,11 +51,14 @@ export const AlertDetailsDocument = gql`
     alert(input: $input) {
       alertId
       ruleId
+      title
```
Comment (3nvi) [inline web/src/pages/AlertDetails/graphql/alertDetails.generated.ts]: same thing, remove whatever you don't use :)

File: web/src/pages/ListAlerts/columns.tsx [hunk 4]
```diff
@@ -29,21 +29,27 @@ import { SEVERITY_COLOR_MAP } from 'Source/constants';
...
+    sortable: true,
+    header: 'Dedup String',
+    flex: '0 0 200px',
+  },
   {
     key: 'eventsMatched',
     sortable: true,
     header: 'Events Count',
-    flex: '1 0 50px',
+    flex: '0 0 120px',
```
Comment (3nvi) [inline web/src/pages/ListAlerts/columns.tsx]: why `120px`? Seems a lot for just a number

If you do itt to fill in the space, then ok

File: web/src/pages/ListAlerts/graphql/listAlerts.generated.ts [hunk 5]
```diff
@@ -48,11 +55,13 @@ export const ListAlertsDocument = gql`
     alerts(input: $input) {
       alertSummaries {
         alertId
+        title
```
Comment (3nvi) [inline web/src/pages/ListAlerts/graphql/listAlerts.generated.ts]: same comment about remooving what's un-neeeded :)

File: web/src/pages/ListAlerts/columns.tsx (hunk 4 above)
Comment (alexmylonas) [reply web/src/pages/ListAlerts/columns.tsx]: reverted

File: web/src/pages/ListAlerts/columns.tsx [hunk 6]
```diff
@@ -29,16 +29,22 @@ import { SEVERITY_COLOR_MAP } from 'Source/constants';
 // The columns that the associated table will show
 const columns = [
   {
-    key: 'id',
+    key: 'ruleId',
     sortable: true,
-    header: 'Alert ID',
-    flex: '0 0 350px',
+    header: 'Rule ID',
```
Comment (kostaspap) [inline web/src/pages/ListAlerts/columns.tsx]: Can you change this to `RULE ORIGIN` to be consistent with the name in the alerts detail page?

**Comment by kostaspap** [comment]
Approving this PR since functionality is there ( I deployed and verified). 
Consider my two suggestions. I didn't take deeper look in the JS code.

File: web/src/pages/ListAlerts/columns.tsx (hunk 6 above)
Comment (alexmylonas) [reply web/src/pages/ListAlerts/columns.tsx]: Done

**Pull request merged**